git-hub

```diff
@@ -1713,17 +1712,17 @@ class RebaseCmd (PullUtil):
         if cls.rebasing() or cls.merging():
             return
         head_ref, head_name = cls.get_ref()
-        if args.action == '--continue' and \
-                cls.get_tmp_ref(args.pull) == head_name:
+        if (args.action == '--continue' and 
+                cls.get_tmp_ref(args.pull) == head_name):
```

Comment:
I am not sure if it is suggested to `()` in this case. Just checking...

Reply:
That's really just a personal preference .. I prefer using `(...)` to split an expression across lines, rather than a backslash.